<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Saddleback College Tour</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>501</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>170</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>170</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>170</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>170</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>170</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>170</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>170</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>170</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>85</red>
          <green>170</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(85, 170, 255)</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="splashPage">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>1</height>
      </size>
     </property>
     <widget class="QLabel" name="headerLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-10</y>
        <width>701</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Bright</family>
        <pointsize>26</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
background-color: rgba(11, 9,117,1);
color: white;
}</string>
      </property>
      <property name="text">
       <string>College Campus Tours</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_mainImage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>701</width>
        <height>411</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="style.qrc">:/qss_icons/rc/Images/ucla.jpg</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="adminLoginButton_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>300</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>131</width>
        <height>32</height>
       </size>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>122</red>
            <green>122</green>
            <blue>122</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>122</red>
            <green>122</green>
            <blue>122</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>122</red>
            <green>122</green>
            <blue>122</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>122</red>
            <green>122</green>
            <blue>122</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>122</red>
            <green>122</green>
            <blue>122</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>122</red>
            <green>122</green>
            <blue>122</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Button">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>122</red>
            <green>122</green>
            <blue>122</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="ButtonText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>122</red>
            <green>122</green>
            <blue>122</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>122</red>
            <green>122</green>
            <blue>122</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family:'Harmonia Sans Pro,Arial,sans-serif';
	font-size: 20px;
	background-color:rgba(122,122,122,255);
	color:white;
	border-radius:10px;</string>
      </property>
      <property name="text">
       <string>Administrator Login</string>
      </property>
     </widget>
     <widget class="QPushButton" name="beginTripButton">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>301</y>
        <width>191</width>
        <height>51</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>113</width>
        <height>32</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family:'Harmonia Sans Pro,Arial,sans-serif';
	font-size: 20px;
	background-color:rgba(122,122,122,255);
	color:white;
	border-radius:10px;</string>
      </property>
      <property name="text">
       <string>Begin Trip</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_transparentBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>270</y>
        <width>451</width>
        <height>121</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">	background-color: rgba(31, 34, 31, .70);
	border-radius: 10px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>headerLabel</zorder>
     <zorder>label_mainImage</zorder>
     <zorder>label_transparentBox</zorder>
     <zorder>beginTripButton</zorder>
     <zorder>adminLoginButton_2</zorder>
    </widget>
    <widget class="QWidget" name="mainMenuPage">
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>661</width>
        <height>281</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout"/>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>541</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
background-color: rgba(11, 9,117,1);
color: white;
}</string>
      </property>
      <property name="text">
       <string>Select an initial college to begin planning your trip from Saddleback:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>640</y>
        <width>501</width>
        <height>32</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="planTripFromSaddlebackButton">
         <property name="text">
          <string>Shortest Trip from Saddleback</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mostEffecientTripFromUCIButton">
         <property name="text">
          <string>Shortest Trip from UCI</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="adminLoginPage">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>190</y>
        <width>321</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="adminLoginLayout">
       <property name="spacing">
        <number>1</number>
       </property>
       <item>
        <widget class="QLabel" name="loginTitleLabel">
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>1000</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
background-color: rgba(255, 0,0,0.25);
color: white;
}</string>
         </property>
         <property name="text">
          <string>Please login:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="loginErrorLabel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>500</width>
           <height>500</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
background-color: rgba(0, 0,0,0.0);
color: red;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="usernameInput">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="passwordInput">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="placeholderText">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="adminLoginButton">
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loginPageBackButton">
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="adminPanelPage">
     <widget class="QFrame" name="CollegeOperationsFrame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>211</width>
        <height>161</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="collegeOperationsLayout">
         <property name="spacing">
          <number>50</number>
         </property>
         <item>
          <widget class="QPushButton" name="newCollegeButton">
           <property name="minimumSize">
            <size>
             <width>131</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>Upload College From File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addNewMenuItemButton">
           <property name="minimumSize">
            <size>
             <width>131</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>Add Menu Item</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="menuItemsLabel">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>0</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
color: yellow;
}</string>
      </property>
      <property name="text">
       <string>Menu Items</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QListWidget" name="menuItemsListWidget">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>140</y>
        <width>391</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>35</y>
        <width>251</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
background: rgba(0,0,0,0);
color:yellow;
}</string>
      </property>
      <property name="text">
       <string>Double click a menu item to modify it</string>
      </property>
     </widget>
     <widget class="QPushButton" name="adminPageBackButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>450</y>
        <width>151</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Back to Main Menu</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="headerLabel_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lucida Bright</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
background-color: rgba(11, 9,117,1);
color: white;
}</string>
    </property>
    <property name="text">
     <string>College Campus Tours</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <action name="actionNew_Trip">
   <property name="text">
    <string>New Trip</string>
   </property>
  </action>
  <action name="adminLoginAction">
   <property name="text">
    <string>Home</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="style.qrc"/>
 </resources>
 <connections/>
</ui>
